db/schema.rb

```diff
@@ -16,20 +16,21 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension 'plpgsql'
 
-  create_table 'applicants', force: :cascade do |t|
-    t.datetime 'created_at', null: false
-    t.datetime 'updated_at', null: false
-    t.string 'utorid'
-    t.string 'student_number'
-    t.string 'first_name'
-    t.string 'last_name'
-    t.string 'email'
-    t.string 'phone'
-    t.text 'address'
-    t.text 'commentary'
-    t.string 'dept'
-    t.integer 'year_in_program'
-    t.string 'full_time'
+  create_table "applicants", force: :cascade do |t|
+    t.datetime "created_at", null: false
```

Comment:
This is probably legacy, but `rubocop` should change these to single quotes

Reply:
oops i forgot to run rubocop manually before commiting. your pre-commit will fix this :)